app/Filament/Resources/CaseStudyResource.php

```diff
@@ -19,7 +19,7 @@
 class CaseStudyResource extends Resource
 {
     protected static ?string $model = CaseStudy::class;
-    protected static ?string $label = "case study";
+    protected static ?string $label = "Case study";
```

Comment:
Why is that?

Reply:
Previous one was better: 
![obraz](https://github.com/blumilksoftware/website/assets/10898728/5f8d0d81-0b06-4a8d-930a-beb9df15649b)

Reply:
fixed and adjusted in other resources